functions/common/function-test-support/src/main/java/org/springframework/cloud/fn/test/support/xmpp/XmppContainerSupport.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2014-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.test.support.xmpp;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.testcontainers.containers.DockerComposeContainer;
+
+import org.springframework.core.io.ClassPathResource;
+
+public abstract class XmppContainerSupport {
```

Comment:
I am sure you noticed the barrage of logging from the testcontainer now when running tests. To get rid of it you can create a `/stream-applications/functions/common/function-test-support/src/main/resources/logback-test.xml` that looks like this: 

```xml
<configuration>
    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
        <encoder>
            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger - %msg%n</pattern>
        </encoder>
    </appender>
    <root level="WARN">
        <appender-ref ref="STDOUT"/>
    </root>
    <logger name="org.testcontainers" level="ERROR"/>
    <logger name="com.github.dockerjava" level="ERROR"/>
</configuration>
```

And voila, its gone.